frontend/src/containers/App.js

```diff
@@ -18,41 +18,105 @@ function App(props) {
   ];
   let roomyPcs = {};
   let myPeerId = "";
-  const [roomVideoStreams, dispatchVideoStreams] = useReducer(addremoveVideoStream, []);
+  const [roomMediaStreams, dispatchMediaStreams] = useReducer(editMediaStream, []);
 
   useEffect(() => {
     rssClient.askToConnect()
   })
 
+  function toggleAudio() {
+    let toggleAudioData = {
+      'action': 'ToggleAudioStream',
+    }
+    dispatchMediaStreams(toggleAudioData)
+  }
+
+  function toggleVideo() {
+    let toggleVideoData = {
+      'action': 'ToggleVideoStream',
+    }
+    dispatchMediaStreams(toggleVideoData)
+  }
+
+  // toggleMediaTrack enables or disables audio or video tracks on a media
+  // stream.
+  function toggleMediaTracks(stream, isAudio) {
+    if (stream === null) {
+      console.log('Invalid video stream, cannot toggle media.')
+      return
+    }
+    var localMediaTracks = stream.getVideoTracks();
```

Comment:
I prefer to use `let` everywhere, so for consistency we should always use `let` unless there's a use case for `var`